modules/osdk-prepare-supported-images.adoc

```diff
@@ -1,56 +0,0 @@
-// Module included in the following assemblies:
-//
-// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
-// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
-// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
-
-:osdk_ver: v1.8.0
-ifeval::["{context}" == "osdk-golang-tutorial"]
-:golang:
-:type: Go
-endif::[]
-
-[id="osdk-prepare-supported-images_{context}"]
-= Preparing your Operator to use supported images
-
-Before running your {type}-based Operator on {product-title}, update your project to use supported images.
-
-.Procedure
-
-ifdef::golang[]
-. Update the project root-level Dockerfile to use supported images. Change the default runner image reference from:
-+
-[source,terminal]
-----
-FROM gcr.io/distroless/static:nonroot
-----
-+
-to:
-+
-[source,terminal]
-----
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
-----
-
-. Depending on the Go project version, your Dockerfile might contain a `USER 65532:65532` or `USER nonroot:nonroot` directive. In either case, remove the line, as it is not required by the supported runner image.
```

Comment:
@emmajiafan Just to confirm, you also meant that this Golang-specific step about the `USER` directive can also be removed, yes?